tests/test_edgeql_explain.py

```diff
@@ -235,25 +235,25 @@ async def test_edgeql_explain_with_bound_01(self):
                                     "type": "relation",
                                     "important": True,
                                 },
-                                {
-                                    "title": "scan_direction",
-                                    "value": "Forward",
-                                    "type": "text",
-                                    "important": True,
-                                },
-                                {
-                                    "title": "index_name",
-                                    "value":
-                                        "index of object type 'default::User' "
-                                        "on (__subject__.name)",
-                                    "type": "index",
-                                    "important": True,
-                                },
-                                {
-                                    "title": "index_cond",
-                                    "type": "expr",
-                                    "important": False,
-                                },
+                                # {
+                                #     "title": "scan_direction",
+                                #     "value": "Forward",
+                                #     "type": "text",
+                                #     "important": True,
+                                # },
+                                # {
+                                #     "title": "index_name",
+                                #     "value":
+                                #         "index of object type 'default::User'"
+                                #         " on (__subject__.name)",
+                                #     "type": "index",
+                                #     "important": True,
+                                # },
+                                # {
+                                #     "title": "index_cond",
+                                #     "type": "expr",
+                                #     "important": False,
+                                # },
```

Comment:
We need to track this down more before merging, right?

Reply:
Yes